### ISSUE TITLE
[Bug] ~Filesize

### ISSUE THREAD

**Issue by githottub** [ErrorFlynn/ytdlp-interface#90]
Version 2.5.0

The approximate file size column is showing values MUCH smaller than the actual file size. My guess is that when it's selecting separate video and audio to satisfy the Best parameter, it is basing the file size on just the audio rather than adding both together.

**Comment by ErrorFlynn** [comment]
Yes, yt-dlp does that (not the interface). If you look at the formats list, the video format yt-dlp selects (marked with `*`) lacks the filesize info, so yt-dlp decides to report just the filesize of the audio format. I guess the interface could mitigate that by detecting that scenario and not displaying the info in that case (I'll see what I can do).
